modules/storages/app/services/storages/nextcloud_managed_folder_create_service.rb

```diff
@@ -309,27 +210,18 @@ def build_set_permissions_input_data(file_id, user_permissions)
       Peripherals::StorageInteraction::Inputs::SetPermissions.build(file_id:, user_permissions:)
     end
 
-    def remote_group_users
-      info "Retrieving users that a part of the #{@storage.group} group"
-      group_users.call(storage: @storage, auth_strategy:, group: @storage.group)
-    end
+    def create_folder = Peripherals::Registry.resolve("nextcloud.commands.create_folder")
 
-    ### Model Scopes
+    def rename_file = Peripherals::Registry.resolve("nextcloud.commands.rename_file")
```

Comment:
❓ why removing the injector and instead defining a method for every command/query used?

Reply:
I kept it at first, but in the end I had a circular dependency that @mereghost spotted as such. I.e. it was a problem that to use the injector you needed to require the registry, but to load the registry you needed to load the create service, which needed to load the injector, which needed to load the registry, which needed to load the create service, which needed to load the injector...

@mereghost shrugged the injector off as an idea/experiment that we could also remove if it caused problems.

Reply:
I see, that was the discussion about circular dependencies. ack.